scripts/k6/redis-lock-test.js

```
@@ -0,0 +1,58 @@
+import http from 'k6/http';
+import {check} from 'k6';
+import {SharedArray} from 'k6/data';
+import {parse} from 'https://jslib.k6.io/papaparse/5.1.1/index.js';
+import {Counter, Trend} from 'k6/metrics';
+
+export let registered = new Counter('registered');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'registered' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)